backend/src/main/java/kr/touroot/travelogue/service/TravelogueService.java

```diff
@@ -36,4 +37,15 @@ public Page<Travelogue> findAll(Pageable pageable) {
     public Page<Travelogue> findAllByMember(Member member, Pageable pageable) {
         return travelogueRepository.findAllByAuthor(member, pageable);
     }
+
+    public void delete(Travelogue travelogue, Member author) {
+        validateAuthor(travelogue, author);
+        travelogueRepository.delete(travelogue);
+    }
+
+    public void validateAuthor(Travelogue travelogue, Member author) {
+        if (!travelogue.isAuthor(author)) {
+            throw new ForbiddenException("작성자만 가능합니다.");
+        }
```

Comment:
```suggestion
        if (!travelogue.isAuthor(author)) {
            throw new ForbiddenException("여행기는 삭제는 작성자만 가능합니다.");
        }
```

조금 더 구체화된 메시지가 좋을 것 같아유~